external/kafka/src/main/scala/org/apache/spark/streaming/kafka/KafkaUtils.scala

```diff
@@ -684,3 +679,53 @@ private class KafkaUtilsPythonHelper {
     kafkaRDD.offsetRanges.toSeq
   }
 }
+
+private object KafkaUtilsPythonHelper {
+  private var initialized = false
+
+  def initialize(): Unit = {
```

Comment:
Oh, I see. You want to call `initialize` to load the object `KafkaUtilsPythonHelper` in the closure. If so, how about adding a method in `KafkaUtilsPythonHelper`, such as
```
object KafkaUtilsPythonHelper {
...
def createPicklerIterator: Iterator[Array[Byte]] = {
  new SerDeUtil.AutoBatchedPickler(iter)
}
...
}
```
and call it in the closure?